src/coreclr/gc/gc.cpp

```diff
@@ -45589,6 +45589,18 @@ void PopulateDacVars(GcDacVars *gcDacVars)
     };
     static_assert(sizeof(gc_heap_field_offsets) == (GENERATION_TABLE_FIELD_INDEX + 1) * sizeof(int), "GENERATION_TABLE_INDEX mismatch");
 #endif //MULTIPLE_HEAPS
+    static int generation_field_offsets[] = {
+
+#define DEFINE_FIELD(field_name, field_type) offsetof(generation, field_name),
+#define DEFINE_DPTR_FIELD(field_name, field_type) offsetof(generation, field_name),
+#define DEFINE_MISSING_FIELD -1,
```

Comment:
instead of defining a missing field to just -1, I would define it like this so it's very easy to see which missing field you are defining -

`#define DEFINE_MISSING_FIELD(field_name) -1,`

Reply:
(sorry this is from the last PR but I didn't get a chance to review before you merged)